src/natcap/invest/recreation/recmodel_client.py

```diff
@@ -42,18 +43,60 @@
 # 'marshal' serializer lets us pass null bytes in strings unlike the default
 Pyro4.config.SERIALIZER = 'marshal'
 
+predictor_table_columns = {
+    "id": {
+        "type": "freestyle_string",
+        "about": ("A unique identifier for the predictor (10 "
+                  "characters or less).")
+    },
+    "path": {
+        "type": {"raster", "vector"},
+        "about": "A spatial file to use as a predictor",
+        "bands": {1: {"type": "number", "units": u.none}},
+        "fields": {},
+        "geometries": {'POINT', 'LINE', 'POLYGON'}
+    },
+    "type": {
+        "type": "option_string",
+        "options": {
+            "raster_mean": ("Predictor is a raster. Metric is the "
+                            "mean of the non-nodata values of the raster that "
+                            "intersect the AOI grid cell or polygon."),
```

Comment:
Oh I think I may have used a mix of lists and dicts for `option_string`s because I couldn't decide which made sense. There are cases like this where each option needs a description, and I think it makes sense to map that to the option. But there are other cases where the options are totally self-descriptive. Maybe they should all use dicts, and if the option doesn't need a description, its value can be an empty string? e.g. this one would stay like it is, and a simpler one would be like: 
```
"options": {
    "yes": "",
    "no": ""
    ...
```
rather than 
```
"options": ["yes", "no]`
```
Then when we go to access the option description, we can fall back to the option name if the description is empty?

Reply:
I switched them all to `dict`s in 7f7b13a, but now I'm leaning the other way... several feel very redundant as a `dict`. See [here](https://github.com/natcap/invest/pull/526/commits/7f7b13a775964da102e5df8b32d75d7b28df8761#diff-a9f288e2be9263f900fb8f8b5e472cd0b3017a0e4144f60f4c8edf770a32ec0fL142-R149), [here](https://github.com/natcap/invest/pull/526/commits/7f7b13a775964da102e5df8b32d75d7b28df8761#diff-87d9293323208d303a9982106696964dc08134882d76bda0ebecbcf7bb87b6cfL48-R54), and [here](https://github.com/natcap/invest/pull/526/commits/7f7b13a775964da102e5df8b32d75d7b28df8761#diff-4712e31b0a9d93a7dc6e24ef43137bf854470777150eeda5c1e434927befc36dL53-R93). Maybe allowing both `dict`s and `set`s would be better.

Reply:
This was worth doing, though, since it prompted me to make descriptions for some options that were missing them. It's pretty tricky to figure out what some of them mean and then give them a concise description.